WellLog.Lib.Test/Business/LasSectionBusinessTests.cs

```diff
@@ -0,0 +1,253 @@
+﻿using Moq;
+using NUnit.Framework;
+using System.IO;
+using System.Linq;
+using System.Text;
+using WellLog.Lib.Business;
+using WellLog.Lib.Enumerations;
+using WellLog.Lib.Exceptions;
+
+namespace WellLog.Lib.Test.Business
+{
+    [TestFixture]
+    public class LasSectionBusinessTests
+    {
+        private Mock<ILasSectionLineBusiness> _lasSectionLineBusiness;
+        private LasSectionBusiness _lasSectionBusiness;
+
+        [SetUp]
+        public void PerTestSetup()
+        {
+            _lasSectionLineBusiness = new Mock<ILasSectionLineBusiness>();
+            _lasSectionBusiness = new LasSectionBusiness(_lasSectionLineBusiness.Object);
+        }
+
+        [Test]
+        public void LasSectionBusiness_ReadMnemonicSection_Pass_NullStream()
+        {
+            Assert.IsNull(_lasSectionBusiness.ReadMnemonicSection(null));
+        }
+
+        [Test]
+        public void LasSectionBusiness_ReadMnemonicSection_Pass_ReadToEndOfStream()
+        {
+            var mnemonicSection = "MNE1.UNIT DATA:DESCRIPTION\r\nMNE2.UNIT DATA:DESCRIPTION\r\n";
+            var lasStream = new MemoryStream(Encoding.ASCII.GetBytes(mnemonicSection));
```

Comment:
This same code is repeated in every test.

Create a private method in your tests to reduce this duplication.

```c#
private Stream _createMemoryStreamFrom(String mnemonicSection)
{
    return new MemoryStream(Encoding.ASCII.GetBytes(mnemonicSection));
}
```

Why?
- Tests are as important as your application code and should read equally as cleanly because they are the documentation for how a program works.

Reply:
done in new pull request